src/fielddef.ts

```diff
@@ -35,6 +35,9 @@ export interface ConditionalValueDef<T> extends ValueDef<T> {
 export interface FieldDef {
   /**
    * Name of the field from which to pull a data value.
+   *
+   *  __Default value:__ `false`
```

Comment:
false?????

Since this is required, there is no default value. 

Add __Required.__ before "Name of ..." and remove default value.